packages/store-ui/src/atoms/Price/Price.stories.mdx

```diff
@@ -0,0 +1,128 @@
+import { Meta, Story, Preview, ArgsTable } from '@storybook/addon-docs/blocks'
```

Comment:
I just have one final thought in the price component. Is there a way of adding styles for each variant? So we can vary the style when we change the variant in the controls pannel. For instance, when selecting the listing price we should see the dashed price 
![image](https://user-images.githubusercontent.com/1753396/123107899-5379f980-d410-11eb-851b-ddec8ab9f08c.png)

Reply:
Sure, @icazevedo can you help me with this? I remember that you showed a dynamic style in the last Demo.

Btw i think you put the wrong print here @tlgimenes lol

Reply:
@Gmantiqueira I did it on code sandbox. I have some of the styles on the [price atom PR] (https://github.com/vtex/faststore/pull/760). That's why I think we should have a better way of dealing with styles on storybook. I think they should be easy to see and to change. I saw something that could help us [here](https://storybook.js.org/blog/storybook-addons-for-css/). In my opinion, we can explore that in other PR.

Reply:
@icazevedo Hmm, nice! I was thinking about putting a CSS-in-JS inside the storybook to handle those dynamic styles only in our stories. We can export only the raw component, but we style them inside our stories. Something like this, so we can make it clear that we have the styled one only inside the story and we export the raw component without the style.

I agree that we can make this in another PR @tlgimenes.